.github/actions/save-node/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Agoric SDK save Node.js build
```

Comment:
9abb3b3aa is a very nice refactor; I'd be inclined to use `refactor(...): ...` rather than `ci(...): ...`.

Unless... does it actually change behavior in some way?

Reply:
It doesn't change any behaviour, but it definitely is only a refactor of the CIish things, not actual package code.  So, I'm on the fence with this one.

Taking the shortest path means leaving it as `ci`.  Hope that's okay.

Reply:
Sure it's fine. Not worth a re-spin on its own, especially since neither ci nor refactor goes in release notes.